docs/plugin-intro.rst

```diff
@@ -19,26 +19,31 @@ Porcupine's API documentation assume
 Where are the plugins?
 ----------------------
 
+At the time of writing this guide, making a new plugin usually requires some
+changes to the rest of Porcupine, so you should start by following
+`the developing instructions in Porcupine's README <https://github.com/Akuli/porcupine#developing-porcupine>`_.
+
 Porcupine searches for plugins in two places. The plugins that come with
-Porcupine are installed along with it, but Porcupine also creates a user-wide
+Porcupine are installed along with it, but Porcupine also creates a user-specific
 ``plugins`` folder in its config directory. You can check where it is using
 Python::
 
     >>> import porcupine.plugins
     >>> print(porcupine.plugins.__path__[0])
     /home/akuli/.config/porcupine/plugins
 
-You can also use ``--print-plugindir`` as shown :source:`here <more_plugins/>`.
-
-Of course, your plugins are probably not in ``/home/akuli/.config/porcupine/plugins``,
-so you need to run the above commands yourself to find out where your plugins
-should go. Rest of this tutorial calls this place **the plugin directory**.
+You can also open Porcupine's about dialog from the *Help* menu.
 
-Your plugin directory is empty by default. The plugins that come with Porcupine
-are installed in a different place::
+Your plugin directory is empty by default.
+The plugins that come with Porcupine are in a different ``plugins`` folder::
 
     >>> print(porcupine.plugins.__path__[1])
-    /home/akuli/.local/lib/python3.7/site-packages/porcupine/plugins
+    /home/akuli/porcu/porcupine/plugins
+
+In the rest of this tutorial, **the plugin directory** means the place where you
+will put your plugin. It can be either of the two ``plugins`` folders.
+If your plugin will be included with Porcupine, then you shouldn't use your
```

Comment:
> If your plugin will be included with Porcupine

Maybe something like "if you want to publish your plugin so others can use it"

Reply:
It's possible to publish a Porcupine plugin without including it in Porcupine, because it's just a Python file. It's fine to develop plugins like that with your user-specific plugin directory.

Reply:
Ah right, so maybe "if you want your plugin to come with Porcupine"? "be a stock plugin"? – something similar.. "be included with porcupine" feels slightly unclear.

Reply:
I like "If you want your plugin to come with Porcupine". Thanks!